compiler/quilt/test/test_install.py

```diff
@@ -95,10 +95,12 @@ def test_install_team_latest(self):
         """
         table_data, table_hash = self.make_table_data()
         file_data, file_hash = self.make_file_data()
-        contents, contents_hash = self.make_contents(table=table_hash, file=file_hash)
```

Comment:
Actually, these are fine; we're using 110 as the max line length. (it's also set in the .pylintrc)